packages/channels/botpress-channel-web/src/views/web/side/index.jsx

```diff
@@ -153,14 +156,14 @@ export default class Side extends React.Component {
     }
 
     return (
-      <span className={style.downloadIcon}>
+      <span className={`${style.downloadIcon} botpress__download-icon`}>
         <i onClick={this.props.downloadConversation}>
           <svg
             version="1.1"
             xmlns="http://www.w3.org/2000/svg"
             width="17"
             height="17"
-            className={style.downloadSVG}
+            className={`${style.downloadSVG} botpress__download-svg`}
```

Comment:
You didn't give the SVG elements their own names in other places, am I right? Consistency is important here

Reply:
@emirotin I don't understand meaning `own names`? What is it?

Reply:
I mean I think for other icons you did give the public class name to the _outer element_ but not to the SVG elements inside, unlike here. We either give it to all of them or none I think.

Reply:
Implemented for all)